app/managers.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+import sqlite3
+
+from models import Actor
+
+
+@dataclass
+class ActorManager:
+    _connection = sqlite3.connect("cinema.db3")
+    table_name = "actors"
+
+    def all(self) -> list:
+        cinema_cursor = self._connection.execute(
+            "SELECT id, first_name, "
+            f"last_name FROM {self.table_name}"

```

Comment:
Use SELECT *
Because if you will change the Actor model you also must change this part of code